src/Filter.tsx

```diff
@@ -0,0 +1,337 @@
+import * as React from "react";
+import { Switch } from "@headlessui/react";
+
+import { Trait, Attribute, Align } from "./Types";
+import { Dropdown } from "./Dropdown";
+
+export interface Traits {
+    Background: string | undefined;
+    Slug: string | undefined;
+    Chest: string | undefined;
+    Mouth: string | undefined;
+    Head: string | undefined;
+    Eyes: string | undefined;
+    Tail: string | undefined;
+    Back: string | undefined;
+    Hands: string | undefined;
```

Comment:
A property which is some type or undefined can be typed just like `Hands?: string;`. The `?` indicates its optional / can be undefined

Please update all the properties in this interface in this way